package.json

```diff
@@ -29,6 +29,7 @@
     "@babel/preset-env": "^7.4.4",
     "@babel/register": "^7.4.4",
     "bcrypt": "^3.0.6",
```

Comment:
you can uninstall bcrypt package since we have bcryptjs and make necessary updates.